CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### [1.0.1](https://github.com/marionettejs/backbone.radio/releases/tag/v1.0.1)
+
+- **Dependency Change**: Bumped Backbone to allow v1.2.2
+
```

Comment:
This should be in a separate commit (the release commit). If you remove it from this commit I'll cut the release